test/e2e/simple_ovs_bridge.go

```diff
@@ -0,0 +1,50 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("NodeNetworkState", func() {
+	var ()
+	Context("when desiredState is configured", func() {
+		Context("with an ovs bridge up with no ports", func() {
+			BeforeEach(func() {
+				updateDesiredState(ovsBrUpNoPorts(bridge1))
+			})
+			AfterEach(func() {
+				updateDesiredState(ovsBrAbsent(bridge1))
+				for _, node := range nodes {
+					interfacesNameForNodeEventually(node).ShouldNot(ContainElement(bridge1))
+				}
+				resetDesiredStateForNodes()
+			})
+			It("should have the linux bridge at currentState", func() {
+				for _, node := range nodes {
+					interfacesNameForNodeEventually(node).Should(ContainElement(bridge1))
+				}
+			})
+		})
+		Context("with an ovs bridge up", func() {
+			BeforeEach(func() {
+				updateDesiredState(ovsBrUp(bridge1))
+			})
+			AfterEach(func() {
+				updateDesiredState(ovsBrAbsent(bridge1))
+				for _, node := range nodes {
+					interfacesNameForNodeEventually(node).ShouldNot(ContainElement(bridge1))
+				}
+				resetDesiredStateForNodes()
+			})
+			It("should have the ovs bridge at currentState", func() {
+				for _, node := range nodes {
+					interfacesForNode(node).Should(ContainElement(SatisfyAll(
+						HaveKeyWithValue("name", bridge1),
+						HaveKeyWithValue("type", "ovs-bridge"),
+						HaveKeyWithValue("state", "up"),
+					)))
+				}
+			})
+		})
```

Comment:
Please also run a test with internal OVS interface. That is needed if someone wants to set an IP address on the bridge. I would say it is a basic scenario. https://github.com/nmstate/nmstate/blob/master/examples/ovsbridge_create.yml

Reply:
Ok, as per @qinqon comment I'll leave it with state so we can change all of them in a follow-up pr.